### PR TITLE
Update SOAP module spec

### PR DESCRIPTION
**Pull request by MadhukaHarith92**
## Checklist
- [ ] Linked to an issue
- [ ] Updated the changelog
- [ ] Added tests
- [ ] Updated the spec
- [ ] Checked native-image compatibility

### CODE REVIEW

File: docs/spec/spec.md [hunk 1]
```diff
@@ -18,83 +18,77 @@ The conforming implementation of the specification is released and included in t
...
+    xml body = xml `<quer:CheckPhoneNumber xmlns:quer="http://ws.cdyne.com/PhoneVerify/query"> 
+         <quer:PhoneNumber>18006785432</quer:PhoneNumber>
+         <quer:LicenseKey>0</quer:LicenseKey>
+      </quer:CheckPhoneNumber>`;
+
+    _ = check soapClient->sendOnly(body);
+}
 ```
+
+## 3. Supported SOAP Versions
```
Comment (ayeshLK) [inline docs/spec/spec.md:91]: This section has not been included into table-of-content.

Comment (MadhukaHarith92) [reply docs/spec/spec.md:92]: Added.

File: docs/spec/spec.md [hunk 2]
```diff
@@ -18,83 +18,87 @@ The conforming implementation of the specification is released and included in t
...
+import ballerina/soap;
+```
+
+### 2.1 Client
+
+The SOAP client can be used to connect to a SOAP service and retrieve data. 
+
+#### 2.1.1 Initializing the Client
+
+The soap:Client init method requires a valid URL and optional configuration to initialize the client.
```
Comment (dilanSachi) [inline docs/spec/spec.md:45]:
```suggestion
The `soap:Client` `init` method requires a valid URL and optional configuration to initialize the client.
```

File: gradle.properties [hunk 3]
```diff
@@ -9,7 +9,7 @@ downloadPluginVersion=4.0.4
 releasePluginVersion=2.6.0
 ballerinaGradlePluginVersion=1.0.0
 
-ballerinaLangVersion=2201.8.0-20230726-145300-b2bdf796
+ballerinaLangVersion=2201.7.0
```
Comment (dilanSachi) [inline gradle.properties:12]: Arent we going to move to java17 with the release?

Comment (MadhukaHarith92) [reply gradle.properties:12]: @gabilang will send a PR

**Review by dilanSachi — APPROVED** [review]
LGTM

**Pull request merged**